actix/CHANGES.md

```diff
@@ -20,6 +20,9 @@
 [#499]: https://github.com/actix/actix/pull/499
 [#501]: https://github.com/actix/actix/pull/501
 
+### Removed
+- Removed `Resolver` actor [#451]
+
```

Comment:
add this to "unreleased"

Reply:
Changes were made in new commit. Thank you for the comment